pages/acknowledgments.tex

```diff
@@ -8,16 +8,16 @@
 
 
 
-\section*{About this rulebook}
-This is the official rulebook of the RoboCup@Home competition \YEAR.
+\section*{About This Rulebook}
```

Comment:
Should `This` really be capitalized?

Reply:
So after looking at the dictionary :book: this should not be capitalise cause its not a major word. Anything like this, the, or, and should not be capitalise.

Reply:
```suggestion
\section*{About this Rulebook}
```